src/main/java/wooteco/subway/admin/controller/LineController.java

```diff
@@ -0,0 +1,94 @@
+package wooteco.subway.admin.controller;
+
+import java.net.URI;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import wooteco.subway.admin.domain.Line;
+import wooteco.subway.admin.dto.LineRequest;
+import wooteco.subway.admin.dto.LineResponse;
+import wooteco.subway.admin.dto.LineStationCreateRequest;
+import wooteco.subway.admin.service.LineService;
+
+@RestController
+@RequestMapping("/lines")
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @GetMapping()
+    public ResponseEntity getLines() {
+        return ResponseEntity.ok().body(LineResponse.listOf(lineService.showLines()));
+    }
+
+    @GetMapping("/detail")
+    public ResponseEntity getLinesDetail() {
+        List<Line> lines = lineService.showLines();
+        List<LineResponse> linesDetail = lines.stream()
+                .map(line -> lineService.findLineWithStationsById(line.getId()))
+                .collect(Collectors.toList());
+        return ResponseEntity.ok().body(linesDetail);
+    }
+
+    @PostMapping()
+    public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
```

Comment:
다이아몬드 연산자를 이용해 어떤 Api가 어떤 Object를 리턴하는지 명시적으로 표현해주는 건 어떨까요?
```suggestion
    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
```

Reply:
궁금한 내용이 있어서 DM보냈습니다!!!